pkg/vars/vars.go

```diff
@@ -1,5 +1,41 @@
 package vars
 
-// GatewayClassControllerName is the unique identifier indicating controller
-// responsible for relevant resources.
-const GatewayClassControllerName = "konghq.com/blixt"
+// -----------------------------------------------------------------------------
+// ControlPlane Constants
+// -----------------------------------------------------------------------------
+
+const (
+	// GatewayClassControllerName is the unique identifier indicating controller
+	// responsible for relevant resources.
+	GatewayClassControllerName = "konghq.com/blixt"
```

Comment:
This should probably stay vendor agnostic barring the move to sigs right?

Reply:
This is what it was already set to it just moved. I think we can just make one big sweeping PR to change the naming once it fully transitions.